### PR TITLE
New package: kdevelop-1.7.3

### PR DESCRIPTION
**Pull request by yopito**
runtime is fine with x64_64/glibc, let's see if cross-building with Travis is fine.
2nd commit (79c73a8) is related to issue #4037 .

### CODE REVIEW

File: srcpkgs/kdevelop/template [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Template file for 'kdevelop'
+pkgname=kdevelop
+version=4.7.3
+revision=1
+
+build_style=cmake
+configure_args="-Wno-dev"
+
+hostmakedepends="cmake qt-qmake automoc4 perl"
```
Comment (Vaelatern) [inline srcpkgs/kdevelop/template]: You can remove `cmake` from this list.

File: srcpkgs/kdevplatform/template [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Template file for 'kdevplatform'
+pkgname=kdevplatform
+version=1.7.3
+revision=1
+
+build_style=cmake
+configure_args="-Wno-dev"
+
+hostmakedepends="cmake qt-qmake automoc4 perl"
```
Comment (Vaelatern) [inline srcpkgs/kdevplatform/template]: Same here.

File: srcpkgs/kdevplatform/template [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+short_desc="Libraries for use by KDE development tools"
+maintainer="yopito <pierre.bourgin@free.fr>"
+license="GPL-2"
+homepage="https://www.kdevelop.org/"
+distfiles="http://download.kde.org/stable/kdevelop/4.7.3/src/${pkgname}-${version}.tar.bz2"
+checksum=195134bde11672de38838f4b341ed28c58042374ca12beedacca9d30e6ab4a2b
+
+kdevplatform-devel_package() {
+	short_desc+="- development files"
+	depends=kdevplatform
```
Comment (Vaelatern) [inline srcpkgs/kdevplatform/template]: `depends="${sourcepkg}>=${version}_${revision}"`

**Comment by yopito** [comment]
Thanks for your expertise and your time. Integrated requested changes in my commits, waiting after Travis now.

**Comment by xtraeme** [comment]
Why did you split /usr/share/doc and /usr/share/locale? please remove those useless subpkgs.

**Comment by yopito** [comment]
It was intended to reduce package size as stated in Manual.md (most of people do not install localization and/or documentation). Anyway, I will unsplit within subpackages ASAP.

**Comment by Vaelatern** [comment]
Splitting those is useful if and only if they are rather large folders on their own.

**Comment by xtraeme** [comment]
Even so, there's no need to split /usr/share/locale. We don't split locales.

**Comment by yopito** [comment]
OK, rebased. Travis CI is fine too.

File: srcpkgs/kdevplatform/template [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+license="GPL-2"
+homepage="https://www.kdevelop.org/"
+distfiles="http://download.kde.org/stable/kdevelop/4.7.3/src/${pkgname}-${version}.tar.bz2"
+checksum=195134bde11672de38838f4b341ed28c58042374ca12beedacca9d30e6ab4a2b
+
+kdevplatform-devel_package() {
+	short_desc+="- development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove "usr/include/kdevplatform/"
```
Comment (xtraeme) [inline srcpkgs/kdevplatform/template]: vmove usr/include

File: srcpkgs/kdevplatform/template [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+		vmove "usr/lib/lib*.so"
+		vmove "usr/lib/cmake/kdevplatform/"
```
Comment (xtraeme) [inline srcpkgs/kdevplatform/template]: vmove usr/lib/cmake

**Comment by yopito** [comment]
My bad, thanks for pointing this. rebased. Travis CI is OK.

**Pull request merged**